python_modules/dagster/dagster/core/definitions/partition.py

```diff
@@ -495,7 +495,9 @@ def run_config_for_partition(self, partition: Partition[T]) -> Dict[str, Any]:
         return copy.deepcopy(self._user_defined_run_config_fn_for_partition(partition))
 
     def tags_for_partition(self, partition: Partition[T]) -> Dict[str, str]:
-        user_tags = copy.deepcopy(self._user_defined_tags_fn_for_partition(partition))
+        user_tags = copy.deepcopy(
+            validate_tags(self._user_defined_tags_fn_for_partition(partition))
```

Comment:
Slick!  This'll make it even easier to add `dagster-k8s/config` tags for Partitions; thank you!